.github/workflows/docs_changes_summary.yml

```diff
@@ -5,14 +5,20 @@ on:
   pull_request_target:
     branches:
       - main
-
+    types:
+      - opened
+      - edited
```

Comment:
`edited` is needed so the workflow is triggered when a developer fixes the PR title to include a breaking change